app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockViewModel.kt

```diff
@@ -434,6 +444,13 @@ data class VaultUnlockState(
     val fido2RequestUserId: String?
         get() = fido2GetCredentialsRequest?.userId ?: fido2CredentialAssertionRequest?.userId
 
+    /**
+     * If the user requires biometrics to be able to unlock the account.
+     */
+    val requiresBiometricsLogin: Boolean
+        get() = !hasMasterPassword &&
+            (vaultUnlockType == VaultUnlockType.MASTER_PASSWORD && isBiometricEnabled)
```

Comment:
This may be a tad easier to read:

```kotlin
get() = when (vaultUnlockType) {
    VaultUnlockType.MASTER_PASSWORD -> !hasMasterPassword && isBiometricEnabled
    VaultUnlockType.PIN -> false
}
   
```